synapse/tests/test_lib_storm.py

```diff
@@ -11,6 +11,17 @@
 
 class StormTest(s_t_utils.SynTest):
 
+    async def test_lib_storm_basics(self):
+        # a catch-all bucket for simple tests to avoid cortex construction
+        async with self.getTestCore() as core:
+
+            with self.raises(s_exc.NoSuchVar):
+                await core.nodes('inet:ipv4=$ipv4')
+
+            # test that runtsafe vars stay runtsafe
+            msgs = await core.stormlist('$foo=bar $lib.print($foo) if $node { $foo=$node.value() }')
```

Comment:
@jnwatson - cisphyx's example is tested here

Reply:
Not sure how I missed that...